doc/man3/EVP_PKEY_new.pod

```diff
@@ -191,11 +196,6 @@ The EVP_PKEY_new_raw_private_key_ex() and
 EVP_PKEY_new_raw_public_key_ex()
 functions were added in OpenSSL 3.0.
 
-The EVP_PKEY_new_CMAC_key() was deprecated in OpenSSL 3.0.
```

Comment:
This line, though, *is* a history line and should stay here.

Reply:
I moved it back, but it loses the context of the "do this instead" part.